submissions/merowing/js-oop/app.js

```diff
@@ -0,0 +1,147 @@
+const FIELD = {
+    WIDTH: 505,
+    HEIGHT: 475,
+    EDGE: 0,
+    CELL_WIDTH: 101,
+    CELL_HEIGHT: 83,
+    WATER_BLOCK_HEIGHT: 83,
+  };
+  
+  const CHARACTER = {
+    POSITION_X: 202,
+    POSITION_Y: 392,
+    STEP: {
+      X: 101,
+      Y: 83,
+    },
+    HEIGHT: 101,
+    SPRITE: "images/char-boy.png",
+  };
+  
+  const ENEMY_OPTIONS = {
+    POSITION: [
+      {
+        X: -101,
+        Y: 60,
+      },
+      {
+        X: -101,
+        Y: 143,
+      },
+      {
+        X: -101,
+        Y: 226,
+      },
+    ],
+    SPEED: {
+      MIN: 200,
+      MAX: 500,
+    },
+    TRIGGER: {
+      LEFT: 30,
+      RIGHT: 30,
+    },
+    WIDTH: 101,
+    SPRITE: "images/enemy-bug.png",
+  };
+  
+  const Enemy = function (x, y, speed) {
+    this.positionX = x;
+    this.positionY = y;
+    this.speed = speed;
+
+    this.player = player;
```

Comment:
A global variable.
The requirements provide a hint to resolution.

Reply:
done